com.unity.render-pipelines.universal/Editor/ShaderPreprocessor.cs

```diff
@@ -232,6 +234,10 @@ bool StripUnusedFeatures(ShaderFeatures features, Shader shader, ShaderSnippetDa
             if (!IsFeatureEnabled(features, ShaderFeatures.ReflectionProbeBoxProjection) && isReflectionProbeBoxProjection)
                 return true;
 
+            bool isReflectionProbeBoxProjectionOff = compilerData.shaderKeywordSet.IsEnabled(m_ReflectionProbeBoxProjectionOff);
+            if (isReflectionProbeBoxProjectionOff && IsFeatureEnabled(ShaderBuildPreprocessor.alwaysEnabledFeatures, ShaderFeatures.ReflectionProbeBoxProjection))
+                return true;
```

Comment:
In line with the above comment, it would be less error prone to have a helper function to deal with the enabled/disabled stripping.

Reply:
I might be missing something, but why we need this new `alwaysEnabledFeatures`? 

Why not just do this:
`if (isReflectionProbeBoxProjectionOff && IsFeatureDisabled(features, ShaderFeatures.ReflectionProbeBoxProjection))`

Reply:
We can only strip the disabled variants if all URP assets have the feature enabled - this guarantees no one will be using the off variant (as long we do not allow for runtime modifications). This is how the features are collected:
`                    _supportedFeatures |= supportedFeatures;
                    _alwaysEnabledFeatures &= supportedFeatures;
`
When a flag is enabled in `supportedFeatures` it means that at least one asset has that feature enabled, but there can also be assets that have it disabled. 
`_alwaysEnabledFeatures` on the other hand guarantees that the flag is enabled on all assets.

Reply:
k, we should also decide to landing patter as my PR will conflict https://github.com/Unity-Technologies/Graphics/pull/4619